### PR TITLE
Remove unused unit test case.

### PR DESCRIPTION
**Pull request by danileng20**

### CODE REVIEW

**Review by pjenys0 — COMMENTED** [review]
Thanks!

**Review by pjenys0 — APPROVED** [review]

File: supernode/services/artworkregister/task_test.go [hunk 1]
```diff
@@ -825,103 +821,3 @@ func TestTaskVerifyPeersSignature(t *testing.T) {
...
-				if err != nil {
-					fmt.Println("err: ", err.Error())
-				}
-				assert.Nil(t, err)
-			}
-		})
-	}
-}
-
-func newTestImageFile() (*artwork.File, error) {
```
Comment (leonidcode20) [inline supernode/services/artworkregister/task_test.go:913]: Shouldn't remove this function. It is still used in other test cases.

Comment (pjenys0) [reply supernode/services/artworkregister/task_test.go:913]: @leonidcode20 : you're missunderstood with newTestImageFile() in walletnode?

Comment (leonidcode20) [reply supernode/services/artworkregister/task_test.go:913]: @pjenys0: ah yes. naming is so confusing.

**Review by leonidcode20 — APPROVED** [review]

**Review by pjenys0 — APPROVED** [review]

**Pull request merged**